fhirstarter/functions.py

```diff
@@ -76,13 +76,9 @@ def make_create_function(
         async def create_async(
             request: Request,
             response: Response,
+            resource: ResourceType,
             _format: str = FORMAT_QP,
             _pretty: str = PRETTY_QP,
-            resource: ResourceType = Body(
```

Comment:
This was the source of the problem. By specifying the request body this way, it makes it optional.